src/pages/workspaces/hooks/useWorkspacesWithSubmissionStats.ts

```diff
@@ -45,7 +45,7 @@ export const useWorkspacesWithSubmissionStats = (): WorkspacesWithSubmissionStat
   useEffect(() => {
     // After the inital load, workspaces are refreshed after deleting a workspace or locking a workspace.
     // We don't need to reload submission stats in those cases.
-    if (workspaces && !submissionStats) {
+    if (workspaces && workspaces.length > 0 && !submissionStats) {
```

Comment:
Does this cause any problems for brand new Terra users who legitimately have 0 workspaces in their list? In that case, we don't need to get the submission stats, so this could be ok.

Reply:
I doubt zero workspaces is ever encountered in _production_, since the workspaces list includes public workspaces.

Reply:
ah, good point about public workspaces.

Reply:
But this is about loading submission stats-- if the user has 0 workspaces, there would be no submission stats to load.